src/browser.ts

```diff
@@ -172,8 +173,12 @@ export class GrpcClient {
     const authHeader = await this.authClient.getRequestHeaders();
     function serviceClientImpl(method, requestData, callback) {
       let cancelController, cancelSignal;
-      if (typeof AbortController !== 'undefined') {
+      if (isBrowser && typeof AbortController !== 'undefined') {
         cancelController = new AbortController();
+      } else {
+        cancelController = new NodeAbortController();
+      }
+      if (cancelController) {
```

Comment:
why do we need an `if` here when `cancelController` will always either be an `AbortController` or a `NodeAbortController` (because of the `if`/`else`)? Will `NodeAbortController()` return `undefined`?

Reply:
`cancelController` will be `undefined` if a browser (`isBrowser() === true`) has no `window.AbortController` (which is possible for older browsers). We still want to do our best in this case.